source/plugins/go/src/oms.go

```diff
@@ -1148,12 +1180,18 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 		logEntrySource := ToString(record["stream"])
 
 		if strings.EqualFold(logEntrySource, "stdout") {
-			if containerID == "" || containsKey(StdoutIgnoreNsSet, k8sNamespace) {
-				continue
+			if containerID == "" { continue }
+			if containsKey(StdoutIgnoreNsSet, k8sNamespace) {
+				if len(StdoutIncludeSystemPodsSet) == 0 { continue }
+				dsName, deploymentName := GetDSNameAndDeploymentNameFromK8sPodName(k8sPodName)
```

Comment:
This will be costly doing for every log record and  also, we will not be covering all other controller types for example if  jobs, cronjobs and statefulset etc. instead we can change from include_system_pods to include_system_namespace_containers and use the container name specified in the configmap against the container name in the log entry.

Reply:
I thought about this and the reason I went with this route is because different pods across namespaces can have the same container name. In that case we might end up collecting logs from system namespaces we do not want. what do you think?